editions/tw5.com/tiddlers/widgets/TestCaseWidget.tid

```diff
@@ -0,0 +1,46 @@
+caption: testcase
+created: 20230406161341763
+modified: 20230406161341763
+tags: Widgets
+title: TestCaseWidget
+type: text/vnd.tiddlywiki
+
+! Introduction
+
+The testcase widget creates an independent subwiki loaded with the specified payload tiddlers and then renders a specified template from within the subwiki.The default template displays a split view with the source tiddlers on the left and the rendered tiddler titled `Output` on the right. It also displays the tiddler titled `Description` as the heading. This makes it possible to run independent tests that also serve as documentation examples.
+
+! Content and Attributes
+
+The content of the `<$testcase>` widget is not displayed but instead is scanned for <<.wlink DataWidget>> widgets that define the payload tiddlers to be included in the test case. The `$:/core` plugin is automatically included in the payload.
+
+|!Attribute |!Description |
+|<<.attr template>> |Optional title of the template used to display the testcase (defaults to $:/core/ui/testcases/DefaultTemplate). Note that custom templates will need to be explicitly added to the payload |
```

Comment:
Mention the other core provided template `$:/core/ui/testcases/RawJSONTemplate` as being useful for troubleshooting `$data` widgets?